chart/sns-connector/README.md

```diff
@@ -0,0 +1,202 @@
+# OpenFaaS Pro SQS Connector
+
+The SNS connector can be used to invoke functions from an AWS SNS messages.
+
+## Prerequisites
+
+- Purchase a license
+
+  You will need an OpenFaaS License
+
+  Contact us to find out more [openfaas.com/pricing](https://www.openfaas.com/pricing)
+
+- Install OpenFaaS
+
+  You must have a working OpenFaaS installed.
+
+## Configure your secrets
+
+- Create the required secret with your OpenFaaS Pro license code:
+
+```bash
+$ kubectl create secret generic \
+    -n openfaas \
+    openfaas-license \
+    --from-file license=$HOME/.openfaas/LICENSE
+```
+
+- Create an AWS credentials secret:
+
+```bash
+$ kubectl create secret generic -n openfaas \
+  aws-sns-credentials --from-file aws-sns-credentials=$HOME/sns-credentials.txt
+```
+
+## Configure ingress
+
+To receive http calls from AWS SNS the callback url has to be publicly accessible.
+
+Install [cert-manager](https://cert-manager.io/docs/), which is used to manage TLS certificates.
+
+You can use Helm, or [arkade](https://github.com/alexellis/arkade):
+
+```bash
+arkade install cert-manager
+```
+
+Install ingress-nginx using arkade or Helm:
+
+```bash
+arkade install ingress-nginx
+```
+
+Create an ACME certificate issuer:
+
+```bash
+export EMAIL="mail@example.com"
+export DOMAIN="sns.example.com"
```

Comment:
I've never seen the Domain in the issuer before.

Is this correct for a HTTP01 solver?